이건우/week7/leetcode1480.py

```diff
@@ -0,0 +1,12 @@
+class Solution(object):
+    def runningSum(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: List[int]
+        """
+        res = []
+        sums = 0
+        for _ in nums:
+            sums += _
+            res.append(sums)
+        return res
```

Comment:
따로 변수를 선언하지 않고 파라미터로 입력받은 nums를 이용해서 누적연산을 하는것이 어떨까요?
하나의 항에 대해 연산을 하면 해당 항에는 그 이전 모든 항의 정보가 누적되니까요. 👍 
```suggestion
        for i in range(1, len(nums)):
            nums[i] += nums[i-1]
        return nums
```

아니면 좀 더 Pythonic하게 작성해, 이렇게도 가능할 것 같아요. :)
```suggestion
        return [sum(nums[:i+1]) for i in range(len(nums))]
```